main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/kyokomi/emoji"
+
+func main() {
+    emoji.Println("Hello, world :pizza:")
```

Comment:
I think, there is no need to duplicate this file in repo's root directory.
Overall good!